cylc-overview.tex

```diff
@@ -11,38 +8,74 @@ \section*{Cylc Tutorial}
 
 \section{Cylc Overview}
 
-Cylc ("silk") is a workflow scheduling tool of particular use in cycling
-environmental forecasting suites. Cylc understands workflows as collections of
-inter-dependent tasks for example the following diagram shows an example
-workflow as seen by cylc.
+Cylc (``silk'') is a workflow engine that can manage continuous workflows of
+cycling (repeating) tasks for applications such as weather and environmental
+forecasting, and climate simulation.  It employs a novel self-organising
+scheduling algorithm that is simple but powerful: in an evolving task pool,
+tasks (which know their own private inputs, outputs, and cycle point) can
+submit their jobs to run once their inputs have been satisfied by the
+completed outputs of other tasks.
+
+A {\em task} is an abstract representation of a {\em job} that runs on a
```

Comment:
I used "job" to cover both scripts and binary programs.  (You wouldn't call the UM a "script" :grin: ).  However, arguably "script" does cover it for cylc, since all jobs are wrapped in a job script.